code/modules/mob/emote.dm

```diff
@@ -16,6 +16,8 @@
 		return
 	for(var/datum/emote/P in key_emotes)
 		if(P.run_emote(src, param, m_type, intentional))
+			var/list/sig_args = list(P, act, m_type, message, intentional)
+			SEND_SIGNAL(src, COMSIG_MOB_EMOTE, sig_args)
```

Comment:
Don't give a list unless you're making use of that list for good reason. Pass in the arguments instead
`SEND_SIGNAL(src, COMSIG_MOB_EMOTE, P, act, m_type, message, intentional)`